### ISSUE TITLE
ARM Build

### ISSUE THREAD

**Issue by sklemmer** [secrethub/secrethub-cli#87]
I would like to use secrethub on arm machines. Currently this gives an exec format error.
With goreleaser this should be trivial right?

```
-bash: /usr/local/bin/secrethub: cannot execute binary file: Exec format error
```

**Comment by SimonBarendse** [comment]
Except for a tiny hiccup (https://github.com/goreleaser/goreleaser/pull/1031) this was indeed trivial. :smile:
We've released [v0.23.0](https://github.com/secrethub/secrethub-cli/releases/tag/v0.23.0) which includes ARM builds. Enjoy!

**Comment by sklemmer** [comment]
Very nice! Thank you 🙂
